src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
```

Comment:
The constants are well-named and describe their purpose effectively. However, the values for rental days that qualify for short and long term discounts are not defined as constants. It would be beneficial to define these as constants to avoid magic numbers and make the code more maintainable.